src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift

```diff
@@ -829,6 +829,78 @@ class ExposureSubmissionCoordinator: NSObject, RequiresAppDependencies {
 		push(topBottomContainerViewController)
 	}
 
+	// MARK: TestCertificateInfo
+
+	private func showTestCertificateInfo() {
+		let testCertificateInfoViewController = ExposureSubmissionTestCertificateInfoViewController(
+			ExposureSubmissionTestCertificateViewModel(
+				testType: .pcr,
+				presentDisclaimer: { [weak self] in
+					let detailViewController = HTMLViewController(model: AppInformationModel.privacyModel)
+					detailViewController.title = AppStrings.AppInformation.privacyTitle
+					detailViewController.isDismissable = false
+					if #available(iOS 13.0, *) {
+						detailViewController.isModalInPresentation = true
+					}
+					self?.push(detailViewController)
```

Comment:
I'm wondering that we don't have this anywhere already, but could be extracted in a separate `func showPrivacyInformation()`

Reply:
not inside of this flow - most times this was added inside an action block of a dynamicCell